.github/scripts/wasm-target-test-build.sh

```diff
@@ -0,0 +1,17 @@
+GIT_ROOT=$(pwd)
+
+cd /tmp
+
+rustup default stable
```

Comment:
Instead of doing this (which may trigger an extra pull of the rust toolchain), I suggest using the same toolchain we use in the project.  This can be achieved by doing this:
```
cp "${GIT_ROOT}/rust-toolchain" .
```
after the `cd foobar`

Reply:
Hmm, I tried this approach.
The issue is that the `rustup` is unable to choose which version of `cargo` to run the next commands, like `cargo new foobar`.
That's why I added this `rustup default stable` line.

In case of `zkevm` CI, there was `cache` step. https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/4996eb6c11fb4eb4bcde7eff55827ecf6cfaaf03/.github/workflows/test-features.yml#L43-L52
I think that is the reason why the `cp "../rust-toolchain" . || true` worked there, but not here.

Reply:
Nevermind the above comment. @ed255 
Here is a commit for improvement. https://github.com/privacy-scaling-explorations/halo2/pull/298/commits/55acc2214dc3e85d3d9b57da96cfd0850d8f009c